src/planscape/config/conditions.json

```diff
@@ -682,7 +682,7 @@
                 },
                 {
                   "display_name": "Biomass Residues",
-                  "data_provider": "CALFIRE",
+                  "data_provider": "Schatz Energy Research Center"
```

Comment:
I believe the automated tests are failing because this needs a comma at the end of the line.